lib/console/console_io.rb

```diff
@@ -0,0 +1,49 @@
+require "console/messages"
+
+class ConsoleIO
+  def initialize(options = {})
+    @input = options[:input] || $stdin
+    @output = options[:output] || $stdout
+    @messages = options[:messages] || Messages.new
+  end
+
+  def clear
+    output.print(`clear`)
+  end
+
+  def puts(message)
+    output.puts(message)
+  end
+
+  def print(message)
+    output.print(message)
+  end
+
+  def read_int_in_range(from, to)
+    val = get_next
```

Comment:
I think it's better not to use shortened variable names... do you mean "value" in this case? Or "valid"?

Reply:
Maybe "move" would be a better name?

Reply:
Good point, will change it to value